src/main/kotlin/com/team4099/robot2023/subsystems/intake/IntakeIO.kt

```diff
@@ -0,0 +1,111 @@
+package com.team4099.robot2023.subsystems.intake
+
+import org.littletonrobotics.junction.LogTable
+import org.littletonrobotics.junction.inputs.LoggableInputs
+import org.team4099.lib.units.base.amps
+import org.team4099.lib.units.base.celsius
+import org.team4099.lib.units.base.inAmperes
+import org.team4099.lib.units.base.inCelsius
+import org.team4099.lib.units.derived.Angle
+import org.team4099.lib.units.derived.degrees
+import org.team4099.lib.units.derived.inDegrees
+import org.team4099.lib.units.inDegreesPerSecond
+import org.team4099.lib.units.perSecond
+
+interface IntakeIO {
+  class IntakeIOInputs : LoggableInputs {
+    var leaderArmPosition: Angle = 0.0.degrees
```

Comment:
related to comment below about the absolute encoder change: we should log the absolute encoder's raw value here, along with its value multiplied by the gear ratio or whatever, so we can see over the course of a match if the NEO encoder is actually drifting